.env.example

```diff
@@ -0,0 +1 @@
+API_URL=https://api.menherabot.xyz/status
```

Comment:
You've exposed the env
.env need to be private
change it urgently

Reply:
This is not private, this is public anyone who accesses the site can see this.

![image](https://user-images.githubusercontent.com/66279500/136106614-c6c9c47c-de65-4c08-9dee-62fa7685bc5d.png)

The .env.example is to help new contributors

Reply:
kkk tá tudo bem, qualquer pessoa consegue esse URL.

o .env.example estava fazendo falta mesmo.